### DISCUSSION TITLE
Moving from uc to sb

### DISCUSSION THREAD

**Discussion by LimitedTime19** [seleniumbase/SeleniumBase#2234] [Q&A] [answered]
options.add_argument(f'--load-extension={",".join(extension_paths)}')
        options.add_argument('--silent-debugger-extension-api')
        options.add_argument(f'--remote-debugging-port={port}')
        
        Hello, I am trying to move my code from uc to sb once and for all however, there's no support for to load multiple extensions with along with the fact that I want to use silent-debugger-extension-api and remote-debugging-port

**Comment by mdmintz** [comment, marked as answer]
Hello. Both undetected-chromedriver and UC Mode do not support zipped/CRX Chrome extensions. See the tickets about it: https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues?q=is%3Aissue+is%3Aopen+crx

But the good news is that if you unzip those extensions into folders first, then you can load them in via command-line option:

https://github.com/seleniumbase/SeleniumBase/blob/466f8d4a25d6c948f40d68b36bac27d46d25d245/help_docs/customizing_test_runs.md?plain=1#L140

View all options here: [SeleniumBase/help_docs/customizing_test_runs.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/customizing_test_runs.md) (Use `--chromium-arg=ARG` for custom Chrome flags.)

As for the remote debugging port, it's already set in UC Mode so that Selenium can control the Chrome process that was spun up first.

**Reply by LimitedTime19** [reply]
Hi, thanks for responding, however I could load one extension like this for example, how do I go about loading 2 or more?

from seleniumbase import SB

with SB(uc=True, extension_dir=extension_path_here) as sb:
    sb.driver.get("https://nowsecure.nl/#relax")
    sb.sleep(1)
    if not sb.is_text_visible("OH YEAH, you passed!", "h1"):
        sb.get_new_driver(undetectable=True)
        sb.driver.get("https://nowsecure.nl/#relax")
        sb.sleep(1)
    if not sb.is_text_visible("OH YEAH, you passed!", "h1"):
        if sb.is_element_visible('iframe[src*="challenge"]'):
            with sb.frame_switch('iframe[src*="challenge"]'):
                sb.click("span.mark")
                sb.sleep(2)
    sb.activate_demo_mode()
    sb.assert_text("OH YEAH, you passed!", "h1", timeout=3)

**Reply by mdmintz** [reply]
Use a comma-separated list for the folders string: Eg.
PATH_TO_FOLDER_1,PATH_TO_FOLDER_2

**Reply by LimitedTime19** [reply]
This below doesn't load any extensions at all now

from seleniumbase import SB

with SB(uc=True, extension_dir=[PATH_TO_FOLDER_1,PATH_TO_FOLDER_2]) as sb:
    sb.driver.get("https://nowsecure.nl/#relax")
    sb.sleep(1)
    if not sb.is_text_visible("OH YEAH, you passed!", "h1"):
        sb.get_new_driver(undetectable=True)
        sb.driver.get("https://nowsecure.nl/#relax")
        sb.sleep(1)
    if not sb.is_text_visible("OH YEAH, you passed!", "h1"):
        if sb.is_element_visible('iframe[src*="challenge"]'):
            with sb.frame_switch('iframe[src*="challenge"]'):
                sb.click("span.mark")
                sb.sleep(2)
    sb.activate_demo_mode()
    sb.assert_text("OH YEAH, you passed!", "h1", timeout=3)

**Reply by mdmintz** [reply]
"PATH1,PATH2"
(Inside one string. Not a Python list.)

**Reply by LimitedTime19** [reply]
you are awesome, issue indeed solved
